application/scss/components/_subnav.scss

```diff
@@ -1,3 +1,57 @@
+//Navigation toggle
+
+.app-subnav-toggle + input {
+  display: none;
+}
+
+.app-subnav-toggle + input:checked + .app-pane__subnav {
+  display: block;
+  @include govuk-media-query(desktop) {
+    display: none;
+  }
+}
+
+.app-subnav-toggle {
+  position: absolute;
```

Comment:
Does this implicitly rely on the container having a `position`?

Reply:
After discussion leaving this as is - element is relative to HTML body due to restrictions of design and layout